### PR TITLE
Creating a model unit test to test the data() functionality.

### PR DESCRIPTION
**Pull request by hub-cap**
- Created factories for instanciating models behind the scenes
- Stubbed out the remote calls to novaclient

### CODE REVIEW

**Comment by cp16net** [comment]
+1

**Comment by hub-cap** [comment]
Moar coverage!!! :)

File: reddwarf/tests/factories/__init__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# vim: tabstop=4 shiftwidth=4 softtabstop=4
+
+# Copyright 2011 OpenStack LLC.
```
Comment (ed-) [inline reddwarf/tests/factories/__init__.py:3]: TIME PARADOX

**Comment by ed-** [comment]
+1

**Pull request merged**